decksite/achievements.py

```diff
@@ -91,6 +93,12 @@ def display(self, p):
         if n > 0:
             return {'name': self.title, 'detail': ngettext(f'1 {self.singular}', f'%(num)d {self.plural}', n)}
         return None
+    def load_summary(self) -> Optional[str]:
+        sql = f"""select sum(`{self.key}`) as count, sum(case when `{self.key}` > 0 then 1 else 0 end) as pcount from _achievements"""
```

Comment:
nit. Please stick to `SELECT SUM(), SUM(CASE WHEN blah > 0 THEN 1 ELSE 0 END) AS pcount FROM ...` style for consistency. 

You might want to avoid using count as a var name as it's a sql reserved word or put it in backticks like we do with the match table.